docker-compose.yml

```diff
@@ -24,7 +24,11 @@ services:
       - .:/app
       - node_modules:/app/node_modules
       - bundle:/usr/local/bundle
-    command: ["rails", "server", "-b", "0.0.0.0"]
+    command:  bash -c "bundle exec rake db:drop &&
```

Comment:
_nipitck_ _suggestion_


I think if we add ,this line it will mean that everytime a developer starts the docker it will drop the database and recreate it. 
In my experience, rarely is this needed, and if someone works over multiple days on a feature could mean it will delete their manual testing data. 

Thus I recommend the following: 

1. Make a script under `./bin/recreate_db` which will execute two commands: 
```
#!/bin/bash

iif [[ "${RAILS_ENV}" == "development" || "${RAILS_ENV}" == "test" ]]; then
  if [[ "${RECREATE_DB}" = true ]]; then
    bundle exec rails db:drop
    bundle exec rails db:prepare
  else
    bundle exec db:migrate
    echo "If you want to drop the database and recreate it set env variable RECREATE_DB to \"true\""
  fi
else
  echo "Unsafe execution of dangerous command in other RAILS_ENV than test or development"
fi
```

This script will only recreate the DB if RAILS_ENV is set to `development` or `test` and if an env var called `RECREATE_DB` is set to `true`. 

2. Add a variable to the environment part of `app` service that will read a variable from host environment in docker
I think something like this should work:
```diff
environment:
  DATABASE_URL: postgres://postgres:password@db:5432/rubyandrails_info_development
+  RECREATE_DB: ${RECREATE_DB}
```

I did not test this so there might be some bugs, but I think the general direction could be this. 

PS: `db:prepare` is equivalent with doing `db:create && db:migrate && db:seed`. See [db:prepare](https://github.com/rails/rails/blob/main/activerecord/lib/active_record/railties/databases.rake#L398) and then [ DatabaseTasks.prepare_all](https://github.com/rails/rails/blob/main/activerecord/lib/active_record/tasks/database_tasks.rb#L173)

Reply:
Hi Lucian, thanks a lot for your suggestions. They were indeed very helpful. 
I changed it so that the default won't drop the database every time.
There is two options in case somebody wants to drop the database, 1) just like you said, setting a environment variable ${RECREATE_DB} to true, 2) access the container, and run "rails db:drop". Both methods worked on my machine. I added that to the Readme as well:

#### Access the shell in the app container
`docker-compose run --rm --entrypoint sh app`

Also changed the seeder to only run if there are no entries of the model.

Reply:
Great work @divagueame 

I will merge this now